src/Breadcrumbs.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import styled from 'styled-components'
+import {COMMON, FLEX, get} from './constants'
+import theme from './theme'
+
+const SELECTED_CLASS = 'selected'
+
+function BreadcrumbBase({className, children, ...rest}) {
```

Comment:
Since we're not pulling `theme` out of the props here we end up stringifying it on the `nav`:

![image](https://user-images.githubusercontent.com/189606/74069146-4347b280-49b2-11ea-9387-46f7d51188e8.png)

Reply:
ooof I forgot about that gotcha - I'll fix it and add it to the contribution guidelines! Nice catch 😎

Reply:
Damn, good eyes @BinaryMuse  👀